### PR TITLE
Fix/cdap ui postmerge

### PR DESCRIPTION
**Pull request by gaarf**
* nail d3 version so that bower doesn't ask on install
 * update package.json and README to reflect new setup

### CODE REVIEW

File: cdap-ui/package.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
   "name": "cdap-ui",
-  "version": "0.1.3",
+  "version": "0.2.8",
```
Comment (nistala) [inline cdap-ui/package.json]: cdap-master is using 2.8.0, @sreevatsanraman should this be the same as cdap version?

**Comment by nistala** [comment]
minor comment, otherwise LGTM

**Comment by elfenheart** [comment]
On the README maybe we want to remove the **npm test** since we don't have a complete test suite yet?

File: cdap-ui/package.json (hunk 1 above)
Comment (gaarf) [inline cdap-ui/package.json]: as of now this version number in package.json does nothing, apart from being shown in the console when the app boots.

**Comment by gaarf** [comment]
@elfenheart why remove an easy way to run the little tests that we do have? FWIW `npm test` will only run karma unit tests, there is no protractor end-to-end anymore.

**Comment by elfenheart** [comment]
:+1: LGTM

**Comment by ajainarayanan** [comment]
:+1: LGTM

**Pull request merged**